internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -65,7 +65,8 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 RUN yum clean all && rm -rf /var/cache/yum/* \
  && yum -y update \
  && yum install -y libffi-devel openssl-devel python36-devel gcc python3-pip python3-setuptools \
- && pip3 install --no-cache-dir --ignore-installed ipaddress \
+ && pip3 install --no-cache-dir \
```

Comment:
Looks like this change didn't make it into https://github.com/openshift/ocp-release-operator-sdk/blob/751d7ef02cd948c4c3591d73e68d7aee9e296c24/release/ansible/upstream.Dockerfile#L28.

@fabianvf, do you make these changes by hand or is there a script? https://github.com/openshift/ocp-release-operator-sdk/commit/f3da6c0518566a59fa8674f07a03a6d1b9195faa

Reply:
a combination of the two, I usually run `cd release/ansible && go run ../../hack/image/ansible/scaffold-ansible-image.go` (same thing for helm, but with the helm scaffolder instead). Might make some untracked files you can ignore, including a Dockerfile which you can then compare to the Dockerfiles in release/ansible.